testhelper/tests.go

```diff
@@ -895,3 +897,43 @@ func (options *TestOptions) setTerraformDir(tempDir string) {
 	options.TerraformDir = tempDir
 	options.WorkspacePath = tempDir
 }
+
+// CheckClusterIngressHealthyDefaultTimeout checks cluster ingress is healthy with given clusterCheckTimeout and clusterCheckdelay
+// clusterId: the ID or name of the cluster
+func (options *TestOptions) CheckClusterIngressHealthyDefaultTimeout(clusterId string) {
+	options.CheckClusterIngressHealthy(clusterId, 10, 1)
+}
+
+// CheckClusterIngressHealthy checks the ingress status of the cluster and returns a boolean based on the status.
+// If the status is "healthy", the test passes. If the status is "Critical" or there is an error, it retries until a timeout is reached.
+// The user can customize the timeout and delay.
+// clusterId: the ID or name of the cluster
+// clusterCheckTimeout: the time until which the ingress status can be checked (in minutes)
+// clusterCheckDelay: the time duration to wait (in minutes)
+func (options *TestOptions) CheckClusterIngressHealthy(clusterId string, clusterCheckTimeout int, clusterCheckDelay int) {
```

Comment:
Should have a suffix for the timeouts to indicate they represent minutes, like `checkTimeoutMins` or `checkTimeoutMinutes`.